custom_components/tuya_local/devices/ferroli_titano_twin.yaml

```diff
@@ -0,0 +1,107 @@
+name: Ferroli Titano Twin
+products:
+  - id: lci0ebsz7ftaruaf
+    name: DT EWH
+primary_entity:
+  entity: water_heater
+  dps:
+    - id: 1
+      name: operation_mode
+      type: boolean
+      mapping:
+        - dps_val: false
+          value: "off"
+        - dps_val: true
+          constraint: work_mode
+          conditions:
+            - dps_val: manual
+              value: electric
+            - dps_val: eco
+              value: eco
+              icon: "mdi:leaf"
+    - id: 2
+      type: string
+      name: work_mode
+      hidden: true
+    - id: 9
+      type: integer
+      name: temperature
+      range:
+        min: 40
+        max: 80
+      mapping:
+        - step: 5
+      unit: C
+    - id: 10
+      type: integer
+      name: current_temperature
+secondary_entities:
+  - entity: switch
+    name: Anti-Legionella
+    category: config
+    dps:
+      - id: 4
+        type: boolean
+        name: switch
+        mapping:
+          - dps_val: true
+            icon: "mdi:shield-cross-outline"
+          - dps_val: false
+            icon: "mdi:shield-remove-outline"
+  - entity: binary_sensor
+    class: problem
+    name: Fault
+    category: diagnostic
+    dps:
+      - id: 101
+        type: bitfield
+        name: sensor
+        mapping:
+          - dps_val: 0
+            value: false
+          - value: true
+            icon: "mdi:water-boiler-alert"
+          - dps_val: 1
+            value: E2 Operating without water
+          - dps_val: 2
+            value: E3 NTC temperature sensor
+          - dps_val: 4
+            value: E4 Water overheating 
+          - dps_val: 16
+            value: E6 clock RTC failure
+  - entity: sensor
+    name: Working status
+    category: diagnostic
+    dps:
+      - id: 102
+        type: string
+        name: sensor
+        mapping:
```

Comment:
If these are the only two options, then this could be a binary_sensor, a suitable class would be "running" (the mapping would need to map to true and false in that case).